src/assets/js/stores/posts.js

```diff
@@ -0,0 +1,29 @@
+import { reactive, toRefs } from "@vue/composition-api";
+import axios from 'axios'
+
+export default function postStore() {
+  const state = reactive({
+    posts: [],
+  })
+
+  async function postGet() {
+    const response = await axios.get('/api/posts')
+    state.posts = response.data
+  }
+
+  async function postDelete(id) {
+    try {
+      const response = await axios.delete(`/api/posts/${id}`);
```

Comment:
if you are not using `response`, no need to assign it?
```suggestion
      await axios.delete(`/api/posts/${id}`);
```

Reply:
your IDE or a linter should point this out, maybe you can try to add eslint? 
Might be overkill for a learning project, but on the other hand a linter is a good learning tool...
WDYT @MasashiYokota ?

Reply:
ESlist config is a bit complex. So, I think you don't have to add eslint